lua/telescope/previewers/buffer_previewer.lua

```diff
@@ -489,6 +488,29 @@ end
 previewers.cat = defaulter(function(opts)
   opts = opts or {}
   local cwd = opts.cwd or vim.loop.cwd()
+  local function jump_to_line(bufnr, winid)
```

Comment:
we inject `entry` already with `lnum` and `col`. why not just use a picker that already has support for jumping to the line and highlighing the line. aka grep_previewer, isnt that good enough?

Reply:
I am here more afraid about potential issues and int the behavior because I might miss because the code is somewhat different and has some handlers that I am not really sure I understand 100%. 

If you 100% sure it will work we can try that out as well